Update index.md Webhooks Actions destination FAQ end to end test (#5101)
segmentio/segment-docs: src/connections/destinations/catalog/actions-webhook/index.md

7. Enable the destination and configured mappings.

{% include components/actions-fields.html settings="true"%}

## FAQs

### Why is a Webhooks (Actions) Destination helpful with end-to-end tests?

> The easiest way to test whether a source's events are sending through the Segment pipeline is with an end-to-end test, triggering the event to send to the Source which sends the event to its connected Destination. By following the steps below you'll understand how to monitor the events arriving to your Segment source and whether they're successfully sending to your destinations. Connecting a Webhooks (Actions) Destination to your sources makes these requests easy to see. For example, if you were to connect a Webhooks Destination (Webhook Actions Destination) to your source, you'd be able to easily see the events received by that source and sent to that destination, proving the events have made it successfully end-to-end.

Reviewer: ```suggestion
The easiest way to test whether a source's events are sending through the Segment pipeline is with an end-to-end test. Use the steps below to monitor the events arriving to your Segment source and whether they're successfully sending to your destinations. Connecting a Webhooks (Actions) Destination to your sources makes these requests easy to see. For example, if you connect a Webhooks Destination (Webhook Actions Destination) to your source, you'd be able to see the events received by that source and sent to that destination.
```

> # Steps to get a Webhook Actions destination connected to your workspace.

Reviewer: ```suggestion
#### Connect a Webhook Actions destination to your workspace
```

> 1. Add a new Webhook (Actions) destination to your source : [link to add Webhooks Actions destination]([url](https://app.segment.com/goto-my-workspace/destinations/catalog/actions-webhook)). Make sure you select the intended source to connect this destination to.

Reviewer: ```suggestion
1. [Add a new Webhook (Actions) destination](https://app.segment.com/goto-my-workspace/destinations/catalog/actions-webhook) to your source. Make sure you select the intended source to connect this destination to.
```

> 2. Visit the Webhook's site, and copy the Webhook's endpoint to clipboard : a simple example site you can use is [https://webhook.site/#!/]([url](https://webhook.site/#!/)), but use whichever Webhooks site you prefer.

Reviewer: ```suggestion
2. Visit the Webhook's site, and copy the endpoint to your clipboard. A simple example site you can use is [https://webhook.site/#!/]([url](https://webhook.site/#!/)), but use whichever Webhooks site you prefer.
```

3. Add a mapping to the Webhook Actions destination, and configure Step 1's conditions to allow for all types of events that you're currently sending through that source.
4. Add the endpoint you copied from Step 2, to the Webhook Actions Mapping's URL in Step 3.
5. Enable the Mapping.
6. Enable the Webhook Actions destination.

> 7. Begin sending events to the Source, and verifying those events throughout the Segment pipeline (source debugger / event delivery), and finally at the webhook's website, which will show the raw JSON for all of the events that were successfully received by your Segment Source and its Webhooks Actions destination.

Reviewer: ```suggestion
7. Begin sending events to your source. Verify those events throughout the Segment pipeline (source debugger/ event delivery), and the webhook's website which shows the raw JSON for all of the events successfully received by your Segment source and its Webhooks Actions destination.
```
